### PR TITLE
Manually clean up Rails' DescendantsTracker in Rails 6

### PR DESCRIPTION
**Pull request by skalee**
I thought that this hack is no longer needed in Rails 6.

Active Support features its own class hierarchy tracker, which was preventing anonymous classes from being garbage collected due to a bug, which has been fixed in Rails 6.0.

For Rails older than 6.0, descendants tracker must be sweeped in `after(:example)` hook.  And it seemed that for Rails 6.0 and newer it is enough to run Ruby's garbage collector to remove classes, which have been created solely for a single RSpec example.

Unfortunately, garbage collector is not guaranteed to work consistently in different Ruby versions, and in fact it was a source of weird build failures in case of Rails 6.0 on Ruby 2.5.7 when `WITHOUT=mongoid` environment variable was set.

In short, manual sweep of descendants tracker is still necessary in Rails 6+.

### CODE REVIEW

**Comment by skalee** [comment]
Oh, and it fixes #86.
